bin/activate.bat

```diff
@@ -4,14 +4,14 @@ for /f "delims=" %%i in ("%~dp0..\envs") do (
     set ANACONDA_ENVS=%%~fi
```

Comment:
I agree that this PR doesn't cover all the features that should be there, it does however fix the ones that are there (and essential, honestly).

Of course I'm not going to do any more work until I'M CONFIDENT SOMEONE WILL REVIEW MY SHIT AT ALL. COME ON. THIS IS JUST RUDE NOW.